pkg/kev/manifest.go

```diff
@@ -119,6 +119,8 @@ func (m *Manifest) CalculateSourcesBaseOverride(opts ...BaseOverrideOpts) (*Mani
 func (m *Manifest) MintEnvironments(candidates []string) *Manifest {
 	fileNameTemplate := m.GetEnvironmentFileNameTemplate()
 
+	symbol := strings.TrimSuffix(ManifestName, filepath.Ext(ManifestName))
```

Comment:
See prior comments regarding `ManifestName` and `symbol`. Additionally, this would be quite useful as a general function: `GetManifestName()`.